zfmt/dag.go

```diff
@@ -130,6 +131,44 @@ func (c *canonDAG) expr(e dag.Expr, parent string) {
 		c.write("type<")
 		c.typ(e.Value)
 		c.write(">")
+	case *dag.RecordExpr:
+		c.write("{")
+		for k, elem := range e.Elems {
+			if k != 0 {
```

Comment:
Nit: I think the intention here is a little clearer with >.
```suggestion
			if k > 0 {
```